developer-docs-site/scripts/additional_dict.txt

```diff
@@ -140,6 +138,8 @@ Labs's
 Leaderboard
 Liveness
 MOV
+MSVC
+MVM
```

Comment:
Is MVM -> MoveVM?

(I'll accept it for now, but let's make sure if it is, to get rid of it and move to `MoveVM`

Reply:
This is not coming from the new Glossary PR and appears long existent:
https://aptos.dev/reference/glossary/#move-virtual-machine-mvm

@movekevin or @wrwg , Do we use MVM?